src/it/java/com/uvocab/api/controller/VocabularyControllerTest.java

```diff
@@ -10,20 +10,21 @@
 import org.springframework.http.HttpMethod;
 import org.springframework.http.HttpStatus;
 import org.springframework.test.context.ActiveProfiles;
-import uvocab.protobuf.v1.Word;
+import uvocab.protobuf.v1.Vocabulary;
 
 @ActiveProfiles({"test"})
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-class WordControllerTest {
+class VocabularyControllerTest {
   @Autowired private TestRestTemplate restTemplate;
 
   @Test
   void shouldpostWord() {
-    var value = Word.newBuilder().setId(1).setValue("Home").build();
+    var word = Vocabulary.newBuilder().setId(1).setWord("Home").build();
```

Comment:
@Danilosimoes why no, `var vocabulary..

Reply:
Cause we're returning a word from vocabulary.

Reply:
return a word or build a Vocabulary ?

Reply:
You are correct!